CommandHelper/LocalCommandHelper/LocalPackages/commands/chars/vegetasupersaiyanbeyondblue.yml

```diff
@@ -0,0 +1,40 @@
+alts:
```

Comment:
The file path is wrong.

You have:
CommandHelper/LocalCommandHelper/LocalPackages/commands/chars/vegetasupersaiyanbeyondblue.yml

What it should be:
CommandHelper/LocalPackages/commands/chars/vegetasupersaiyanbeyondblue.yml